src/main/java/TextView.java

```diff
@@ -102,6 +113,11 @@ void drawBoard(Board board)
 				}
 			}
 		}
+		System.out.print("  ");
+		for (int i =0; i < board.GRID_SIZE; i++){
+			System.out.print("¯¯¯¯");
+		}
+		System.out.println("  ");
 	}
 
 	/**
```

Comment:
- [ ] Replace `getAbbreviation()` with a map from `PieceType` to `String` as an attribute of the class. Build this map in the constructor.